docs/index.ejs

```diff
@@ -13,6 +13,7 @@
 <body>
   <div id="app"><%- html %></div>
   <link rel="stylesheet" href="app.css" />
+  <div data-style-loader></div>
```

Comment:
Now that we're lazy loading CSS, we need to ensure that styles are injected into the DOM *after* the style sheet.